gaffer-core/operation/src/test/java/gaffer/operation/impl/add/AddElementsTest.java

```diff
@@ -57,8 +56,7 @@
             "    \"group\" : \"edge type 2\",%n" +
             "    \"source\" : \"source vertex 1\",%n" +
             "    \"destination\" : \"dest vertex 1\",%n" +
-            "    \"directed\" : true,%n" +
-            "    \"class\" : \"gaffer.data.element.Edge\"%n" +
```

Comment:
When elements are serialised into JSON they should always have the "class" property, so consumers of the API can check if the element is an Edge or an Entity. Why has it been removed from the JSON here?